packages/blade/src/tokens/theme/paymentTheme.ts

```diff
@@ -1009,7 +1033,7 @@ const colors: ColorsWithModes = {
           highContrast: globalColors.neutral.blueGrayDark[600],
         },
         level3: {
-          lowContrast: globalColors.neutral.blueGrayDark[800],
+          lowContrast: globalColors.neutral.blueGrayDark[900],
```

Comment:
Why is this pointing to `blueGrayDark`? Is this meant for the dark theme inside payments?

Reply:
The parent is `onDark` so I think it should be. It was always `blueGrayDark` but the shades are changing, wanted to confirm those.